appium-tests/common/common.spec.js

```diff
@@ -133,7 +136,7 @@ describe('Contacts Android', function () {
     function renameContact(oldName, newGivenName, newFamilyName) {
         return driver
             .context(webviewContext)
-            .setAsyncScriptTimeout(4 * MINUTE)
+            .setAsyncScriptTimeout(7 * MINUTE)
```

Comment:
Right now updating a contact on our osx-android builder takes a long time (around 5 minutes). 
As it is a minor tweak I thought it's OK to include it with another commit.
I'm not sure though that it is a good idea to just increase timeout instead of investigating what's wrong with the simulator on the slave.